### DISCUSSION TITLE
URLs in WLSg apps

### DISCUSSION THREAD

**Discussion by kchan101** [microsoft/wslg#535] [Q&A]
I have managed to install Fedora and get evolution to run surprisingly easily :)

Is there a way to configure evolution (say) to open a native Windows browser when I click on a URL? My current workaround is to have a browser installed in the Fedora instance

**Comment by onomatopellan** [comment]
Take a look at https://github.com/wslutilities/wslu specially `wslview`

**Reply by kchan101** [reply]
Brilliant, thanks!
